packages/non-core/coffeescript/package.js

```diff
@@ -1,29 +1,29 @@
 Package.describe({
-  summary: "Javascript dialect with fewer braces and semicolons",
-  version: "1.12.6_1"
+  name: 'coffeescript',
+  summary: 'Javascript dialect with fewer braces and semicolons',
+  // This package version should track the version of the `coffeescript-compiler`
+  // package, because people will likely only have this one added to their apps;
+  // so bumping the version of this package will be how they get newer versions
+  // of `coffeescript-compiler`.
+  version: '1.12.7_1'
 });
 
 Package.registerBuildPlugin({
-  name: "compileCoffeescript",
-  use: ['caching-compiler', 'ecmascript'],
-  sources: ['plugin/compile-coffeescript.js'],
-  npmDependencies: {
-    "coffeescript": "1.12.6",
-    "source-map": "0.5.6"
-  }
+  name: 'compile-coffeescript',
+  use: ['caching-compiler@1.1.9', 'ecmascript@0.8.2', 'coffeescript-compiler@1.12.7_1'],
```

Comment:
For the record, this means an application developer who's using `coffeescript` can use any version of `caching-compiler` that's `>= 1.1.9` and has the same major version (`1.x.y`). Likewise, any version of `ecmascript` is fair game if it starts with a `0.` and is more recent than `0.8.2`.

I would suggest that we make the `coffeescript-compiler` version constraint exact (`@=1.12.7_1`), unless you think users might ever want to use different versions of `coffeescript` and `coffeescript-compiler`?